src/NotifyFunction/NotifyFunction.csproj

```diff
@@ -10,7 +10,7 @@
     <PackageReference Include="Azure.Communication.Sms" Version="1.0.0-beta.2" />
```

Comment:
There is a GA version of this package - [Azure.Communication.Sms](https://www.nuget.org/packages/Azure.Communication.Sms/). Can we update to 1.0.1?

Reply:
Class `Azure.Communication.PhoneNumber` does not exist in GA version.

Reply:
Looks like we would just pass the phone number as a string - https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/communication/Azure.Communication.Sms#send-a-11-sms-message